### PR TITLE
DO NOT MERGE: RHOARDOC-1670: New branch with changes to replace reverted PR

### PR DESCRIPTION
**Pull request by huntley**
@tradej @inoxx03 The reverted PR for RHOARDOC-1670 can be closed as this PR has all of the changes.

### CODE REVIEW

**Comment by inoxx03** [comment]
[test]

**Comment by huntley** [comment]
@inoxx03 I think I have fixed the merge conflicts, though it took a bit more wrangling than I expected. Can you run the PR build and see if it checks out?

**Comment by inoxx03** [comment]
[test]

File: docs/topics/con_purpose-of-the-relational-database-backend-pattern.adoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+This updates the name of the item with the given ID to match the name specified in your request.
+* Execute an HTTP `DELETE` request on the `api/fruits/*` endpoint, passing in a valid ID as an argument.
+This removes the item with the specified ID from the database and returns an HTTP code `204` (No Content) as a response.
+If you pass in an invalid ID, the call results in an HTTP error `404`.
+
+ifndef::built-for-nodejs[]
+This example also contains a set of automated xref:running-the-example-application-integration-tests_{context}[integration tests] that can be used to verify that the application is fully integrated with the database.
+endif::built-for-nodejs[]
+
+This example does not showcase a fully matured RESTful model (level 3), but it does use compatible HTTP verbs and status, following the recommended HTTP API practices.
```
Comment (akoniar) [inline docs/topics/con_purpose-of-the-relational-database-backend-pattern.adoc]: This sentence is present in a section [3.3.1. Purpose of the relational database backend pattern](https://deploy-preview-1332--launcher-documentation.netlify.com/docs/vertx-runtime.html#purpose-of-the-relational-database-backend-pattern_crud) twice. Please could you remove it from here or form docs/topics/assembly_crud-example-application.adoc.

**Comment by inoxx03** [comment]
This PR can not be completed due to low priority and lack of team resources. It will be closed and  can be reopened later.

**Pull request closed without merging**